README.md

```diff
@@ -89,13 +89,14 @@ s := sanitizer.New(sanitizer.OptionDateFormat{
 
 1. **max=`<n>`** - Maximum string length. It will truncate the string to `<n>` characters if this limit is exceeded
 1. **trim** - Remove trailing spaces left and right
+1. **trim=`<c>`** - Remove trailing characters `<c>` left and right. You can provide more than one character. Example: `trim= \n` will trim whitespaces and new lines
```

Comment:
Nit:

I think we should use the terminology "space" for this character (not "whitespace") because whitespace generally refers to any kind of spacing, both horizontal and vertical. See https://en.wikipedia.org/wiki/Whitespace_character. In the article, they even use the terminology "space" for the character in the table of unicode characters.